cmd/helm/dependency.go

```diff
@@ -71,6 +71,16 @@ the dependency charts stored locally. The path should start with a prefix of
 If the dependency chart is retrieved locally, it is not required to have the
 repository added to helm by "helm add repo". Version matching is also supported
 for this case.
+
+Starting from 3.3.0, if OCI Registry support has been enabled via the HELM_EXPERIMENTAL_OCI
+flag, repository can be defined as an OCI image reference. The path should start with a
+prefix of "oci://". For example,
+
+    # Chart.yaml
+    dependencies:
+    - name: nginx
+      version: "1.2.3"
+      repository: "oci://nginx:latest"
```

Comment:
can you update this to use a proper helm chart URL. `nginx:latest` is a docker container, not a chart. Thanks

Reply:
Add a comment about this in the review also: https://github.com/helm/helm/pull/7613#pullrequestreview-433406051

Reply:
Yep I brought it up in https://github.com/helm/helm-www/issues/637 after chatting with @hickeyma, I'll adjust that in this PR too